src/modules/authentication/components/Verification/Verification.tsx

```diff
@@ -1,20 +1,37 @@
 import styles from './verify.module.css'
 import logo from '../../../../assets/images/PMS 3.svg'
 import { useForm } from 'react-hook-form'
-import { VerifyFormFields } from '../../../../types/types'
+import { useLocation, useNavigate } from 'react-router-dom'
+import { VerifyRegister } from '../../../../interface/AuthResponse/AuthResponse'
+import { AUTH_URLS, axiosInstance, BASE_AUTH } from '../../../../api'
+import { toast } from 'react-toastify'
 
 export default function Verification() {
 
-  const{ 
-    formState:{errors,isSubmitting },
-    register  ,
-    handleSubmit,
-    
-  }=	useForm<VerifyFormFields> ({ mode:'onChange'})
-  const onSubmit = (data:VerifyFormFields)=>{
-console.log(data);
-
-  }
+	const location = useLocation()
+	const myLocation = location.state
+	const navigate = useNavigate()
+	
+	const{ 
+	  formState:{errors,isSubmitting },
+	  register  ,
+      setError,
+	  handleSubmit,
+	
+	  
+	}=	useForm<VerifyRegister> ({defaultValues : {email : myLocation} , mode:'onChange'})
+	const  onSubmit = async(data:VerifyRegister)=>{
+	await axiosInstance.put(BASE_AUTH +AUTH_URLS.verify , data).then((response)=>{
```

Comment:
We can simplify this by including the base auth in the AUTH_URLS

Reply:
done